src/work-allocation/containers/task-list/task-list.component.ts

```diff
@@ -143,19 +143,18 @@ export class TaskListComponent implements OnChanges {
    * TODO: Think about moving 'none' to task sort model.
    *
    * @param fieldName - 'caseReference'
-   * @return 'none' / 'ascending' / 'descending'
+   * @return 'none' / 'asc' / 'desc'
    */
-  public isColumnSorted(fieldName: string): TaskSort {
+  public isColumnSorted(fieldName: string): string {
     // If we don't have an actual sortedBy value, default it now.
     if (!this.sortedBy) {
       const { defaultSortFieldName, defaultSortDirection } = this.taskServiceConfig;
       this.sortedBy = { fieldName: defaultSortFieldName, order: defaultSortDirection };
     }
 
-
     // If this is the field we're sorted by, return the appropriate order.
     if (this.sortedBy.fieldName === fieldName) {
-      return this.sortedBy.order;
+      return this.sortedBy.order === TaskSort.ASC ? 'ascending' : 'descending'
```

Comment:
The return values don't tally with the change to the method description.

Reply:
Agreed, e.g. could change to getColumnSortedSetting

Reply:
Done

Reply:
My comment was to do with the return values - in the description, it says 'asc' and 'desc' but what it actually returns is 'ascending' and 'descending'. Considering the description was changed, shouldn't the values be? If not, why was the description changed? They should match.

Reply:
Changed the values in the description. The return values need to be 'ascending'/'descending' not 'asc'/'desc' as myself and Kuda discovered